main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"log"
+	"pace-sender/pkg/email"
+)
+
+func main() {
+	config, err := email.InitEmailSender()
+	if err != nil {
+		log.Fatal("Error loading email configuration: ", err)
+	}
```

Comment:
Right now if we look at pkg/email/send.go and main.go, we can see that configuration init is duplicated
```suggestion
	config, err := configuration.Load()
	if err != nil {
		return nil, fmt.Errorf("could not load config: %v", err)
	}
	return config, nil
```